Automattic/wp-calypso
# We updated the storage space last year but not this instance

We got a report by @peterschimke that the ecommerce flow shows incorrect space for the space trial.

Looks like we updated the space for both Business and Commerce last year but this particular feature in Calypso remained unchanged.

Related to #

## Proposed Changes

* Fix the space feature for Commerce

## Why are these changes being made?


* The Commerce Trial is inaccurately stating that we offer 200 GB of space for new subscribers. This definition is not used on the plans grid, which render a dropdown with the correct 50 GB of space.

## Testing Instructions

* Start a signup with /setup/entrepreneur/start?ref=woo-hosting-solutions-lp
* Go to step 3 and observe:

<img width="390" alt="Screenshot 2024-11-09 at 0 40 59" src="https://github.com/user-attachments/assets/fae546e1-c287-45dc-aa71-862ce8f81382">
<img width="390" alt="Screenshot 2024-11-09 at 0 40 42" src="https://github.com/user-attachments/assets/67a03ef2-4458-4f86-a6ba-a8665701b3a8">

## Pre-merge Checklist
